libraries/scripting/dependencies-maven/test/kotlin/script/experimental/test/MavenResolverTest.kt

```diff
@@ -81,6 +84,22 @@ class MavenResolverTest : ResolversTestBase() {
         }
     }
 
+    // Ignored - tests with custom repos often break the CI due to the caching issues
+    // TODO: find a way to enable it back
+    @Ignore
+    fun ignore_testAuth() {
+        val resolver = MavenDependenciesResolver()
+        val options = buildOptions(
```

Comment:
It may be also a good idea to add a test with options parsing, like here - https://github.com/JetBrains/kotlin/blob/master/libraries/tools/kotlin-main-kts-test/testData/resolve-error-hamcrest-via-junit.main.kts.

Reply:
I don't know such repositories :(

Reply:
`kotlin.script.experimental.dependencies.impl.SimpleExternalDependenciesResolverOptionsParser` is `internal`, may I make it public?

Reply:
yes, see no probs.